.mailmap

```diff
@@ -0,0 +1,6 @@
+Claire Cadman <claire.cadman@puppet.com> claire cadman <claire.cadman@puppet.com>
+Claire Cadman <claire.cadman@puppet.com> clairecadman <claire.cadman@puppet.com>
+Heston Hoffman <hestonhoffman@gmail.com> hestonhoffman <hestonhoffman@gmail.com>
+<morgan@puppet.com> <morgan@puppetlabs.com>
```

Comment:
I have (oh so impressively) identified this line as different than the rest. Is this right?

Reply:
Yeah, I think so?  Check out `git-blame(1)`.  See the "MAPPING AUTHORS" section at the end.  There are multiple formats depending on what you're trying to normalize.